internal/scheduler/jobdb/gang.go

```diff
@@ -1,5 +1,15 @@
 package jobdb
 
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/pkg/errors"
+
+	"github.com/armadaproject/armada/internal/scheduler/interfaces"
+	"github.com/armadaproject/armada/internal/server/configuration"
+)
+
 type GangInfo struct {
 	isGang         bool
```

Comment:
I think we could now get rid of this and define `isGang()` just to return true if cardinality > 1